build.gradle.kts

```diff
@@ -17,6 +26,30 @@ dependencies {
     testImplementation("io.mockk:mockk:1.13.8")
 }
 
+tasks.withType<Jar> {
+    manifest {
+        attributes(
+            "Main-Class" to "app.MainKt"
+        )
+    }
+}
+
```

Comment:
Creating a fat JAR (which includes dependencies) and the tasks.withType<Jar> block might be redundant. It would be better to consolidate these.